modules/distribution/src/repository/resources/dbscripts/apimgt/mysql.sql

```diff
@@ -203,21 +205,21 @@ CREATE TABLE IF NOT EXISTS IDN_AUTH_SESSION_STORE (
 CREATE TABLE IF NOT EXISTS SP_APP (
         ID INTEGER NOT NULL AUTO_INCREMENT,
         TENANT_ID INTEGER NOT NULL,
-            APP_NAME VARCHAR (255) NOT NULL ,
-            USER_STORE VARCHAR (255) NOT NULL,
+	    	APP_NAME VARCHAR (255) NOT NULL ,
+	    	USER_STORE VARCHAR (255) NOT NULL,
         USERNAME VARCHAR (255) NOT NULL ,
         DESCRIPTION VARCHAR (1024),
-            ROLE_CLAIM VARCHAR (512),
+	    	ROLE_CLAIM VARCHAR (512),
         AUTH_TYPE VARCHAR (255) NOT NULL,
-            PROVISIONING_USERSTORE_DOMAIN VARCHAR (512),
-            IS_LOCAL_CLAIM_DIALECT CHAR(1) DEFAULT '1',
-            IS_SEND_LOCAL_SUBJECT_ID CHAR(1) DEFAULT '0',
-            IS_SEND_AUTH_LIST_OF_IDPS CHAR(1) DEFAULT '0',
+	    	PROVISIONING_USERSTORE_DOMAIN VARCHAR (512),
+	    	IS_LOCAL_CLAIM_DIALECT CHAR(1) DEFAULT '1',
+	    	IS_SEND_LOCAL_SUBJECT_ID CHAR(1) DEFAULT '0',
+	    	IS_SEND_AUTH_LIST_OF_IDPS CHAR(1) DEFAULT '0',
         IS_USE_TENANT_DOMAIN_SUBJECT CHAR(1) DEFAULT '1',
         IS_USE_USER_DOMAIN_SUBJECT CHAR(1) DEFAULT '1',
-            SUBJECT_CLAIM_URI VARCHAR (512),
-            IS_SAAS_APP CHAR(1) DEFAULT '0',
-            IS_DUMB_MODE CHAR(1) DEFAULT '0',
+	    	SUBJECT_CLAIM_URI VARCHAR (512),
+	    	IS_SAAS_APP CHAR(1) DEFAULT '0',
+	    	IS_DUMB_MODE CHAR(1) DEFAULT '0',
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04